### DISCUSSION TITLE
I built a express api boilerplate with mongoose. It's open source and open to contribution

### DISCUSSION THREAD

**Discussion by Mohan1203** [community/community#118660] [Programming Help]
### Select Topic Area

Show & Tell

### Body

I am a Node developer and I use Express for routing. The main problem with Express with that you have to start from scratch every time so decided to create a basic boilerplate for express. A boilerplate has some files and features. The features are authentication middleware, error handling middleware, centralized error handling, JWT token generation and token verification and user login and register route. I use Mongo as database and Mongoose as ORM so you have create MongoDB account and create dotenv file and create variable call DATABASE_URI and SERVER_PORT also create TOKEN_SECRET after you can use this bioler plate and want to change something feel free to do that

[Github repository link:](https://github.com/Mohan1203/express_api_boilerplate)

just use
`git clone https://github.com/Mohan1203/express_api_boilerplate` to clone repository

**Comment by ettaboyle** [comment]
Thanks for posting in the GitHub Community, @Mohan1203!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. **The Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions).** 

This question should be in the [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) category. I've gone ahead and moved it for you. Good luck!
